backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/GeneralViewDiagramDescriptionProvider.java

```diff
@@ -86,11 +86,36 @@ public class GeneralViewDiagramDescriptionProvider implements IRepresentationDes
             Map.entry(SysmlPackage.eINSTANCE.getPartDefinition(),        List.of(SysmlPackage.eINSTANCE.getDefinition_OwnedAttribute(), SysmlPackage.eINSTANCE.getDefinition_OwnedPort())),
             Map.entry(SysmlPackage.eINSTANCE.getPortDefinition(),        List.of(SysmlPackage.eINSTANCE.getDefinition_OwnedAttribute(), SysmlPackage.eINSTANCE.getDefinition_OwnedPort(), SysmlPackage.eINSTANCE.getDefinition_OwnedReference())),
             Map.entry(SysmlPackage.eINSTANCE.getAttributeUsage(),        List.of(SysmlPackage.eINSTANCE.getUsage_NestedAttribute(), SysmlPackage.eINSTANCE.getUsage_NestedReference())),
+            Map.entry(SysmlPackage.eINSTANCE.getEnumerationUsage(),      List.of(SysmlPackage.eINSTANCE.getEnumerationUsage_EnumerationDefinition())),
             Map.entry(SysmlPackage.eINSTANCE.getInterfaceUsage(),        List.of(SysmlPackage.eINSTANCE.getUsage_NestedAttribute(), SysmlPackage.eINSTANCE.getUsage_NestedPort())),
             Map.entry(SysmlPackage.eINSTANCE.getItemUsage(),             List.of(SysmlPackage.eINSTANCE.getUsage_NestedAttribute(), SysmlPackage.eINSTANCE.getUsage_NestedReference())),
             Map.entry(SysmlPackage.eINSTANCE.getPartUsage(),             List.of(SysmlPackage.eINSTANCE.getUsage_NestedAttribute(), SysmlPackage.eINSTANCE.getUsage_NestedPort())),
             Map.entry(SysmlPackage.eINSTANCE.getPortUsage(),             List.of(SysmlPackage.eINSTANCE.getUsage_NestedAttribute(), SysmlPackage.eINSTANCE.getUsage_NestedReference()))
             );
+    
+    public static final Map<String, List<EClass>> TOOL_SECTIONS = Map.ofEntries(
+            Map.entry("Structure", List.of(
+                    SysmlPackage.eINSTANCE.getAttributeUsage(),
+                    SysmlPackage.eINSTANCE.getAttributeDefinition(),
+                    SysmlPackage.eINSTANCE.getEnumerationUsage(),
+                    SysmlPackage.eINSTANCE.getEnumerationDefinition(),
+                    SysmlPackage.eINSTANCE.getItemUsage(),
+                    SysmlPackage.eINSTANCE.getItemDefinition(),
+                    SysmlPackage.eINSTANCE.getPackage(),
+                    SysmlPackage.eINSTANCE.getPartUsage(),
+                    SysmlPackage.eINSTANCE.getPartDefinition()
+                    )),
+            Map.entry("Interconnection", List.of(
+                    SysmlPackage.eINSTANCE.getInterfaceUsage(), 
+                    SysmlPackage.eINSTANCE.getInterfaceDefinition(), 
+                    SysmlPackage.eINSTANCE.getPortUsage(), 
+                    SysmlPackage.eINSTANCE.getPortDefinition()
+                    )),
+            Map.entry("Extension", List.of(
+                    SysmlPackage.eINSTANCE.getMetadataUsage(),
```

Comment:
If you add MetadataUsage, you should also add it in the USAGE list above.
Furthermore, are you sure to fully handled it in this PR? MetadataUsage has a specific graphical notation, which doesn't seem to be handled here.

Reply:
Agree. I've removed `MetadataUsage` for this PR.